invenio_deposit/config.py

```diff
@@ -33,7 +33,7 @@
 DEPOSIT_SEARCH_API = '/api/deposits'
 """URL of search endpoint for deposits."""
 
-DEPOSIT_RECORDS_API = '/api/deposits/{pid_value}'
```

Comment:
The variable is used here: https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/templates/invenio_deposit/edit.html#L48

it looks like jinja doesn't support it because the `{pid_value}` is not replaced by jinja `format()` filter.
See the example here: http://jinja.pocoo.org/docs/dev/templates/#i18n
```
{{ _('Hello %(user)s!')|format(user=user.username) }}
```